src/edu/csus/ecs/pc2/exports/ccs/CLICSEventFeedExract.java

```diff
@@ -0,0 +1,415 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.exports.ccs;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.FileUtilities;
+import edu.csus.ecs.pc2.core.ParseArguments;
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.imports.clics.EventFeedLoader;
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+import edu.csus.ecs.pc2.core.list.TeamAccountComparator;
+import edu.csus.ecs.pc2.core.model.ContestInformation;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.imports.ccs.ContestSnakeYAMLLoader;
+import edu.csus.ecs.pc2.shadow.IRemoteContestAPIAdapter;
+import edu.csus.ecs.pc2.shadow.RemoteContestAPIAdapter;
+
+/**
+ * Extract data from event feed, output data to file
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSEventFeedExract {
```

Comment:
Please correct the spelling of this class name ("...Extract", not "...Exract").

Reply:
fixed name/typo

Reply:
Hmmm...I'm not seeing any change.  Maybe I'm using GitHub's facilities wrong...

In any case, if the requested change has been made, please go ahead and mark this comment "resolved".